microsoft/testsuites/cvm/cvm_attestation.py

```diff
@@ -61,6 +62,10 @@ def verify_azure_cvm_attestation_report(
         result: TestResult,
         variables: Dict[str, Any],
     ) -> None:
+        if node.tools[Lscpu].get_cpu_type() != CpuType.AMD:
```

Comment:
maybe move this condition check to the `before_case` function above, that's where the current environment check is happening

Reply:
Good catch :) Changed